scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -17,7 +17,7 @@ export UPGRADED_CAPM3_VERSION="v1beta1"
 export IMAGE_OS="ubuntu"
 export FROM_K8S_VERSION="v1.23.2"
```

Comment:
Does this maybe need to be updated to 23.4 to make it (latest version - 1) since this var is not used in the node image variable structuring?

Reply:
Done